src/main/java/com/PBA/budgetservice/controller/BudgetController.java

```diff
@@ -0,0 +1,47 @@
+package com.PBA.budgetservice.controller;
+
+import com.PBA.budgetservice.persistance.model.dtos.IncomeRequest;
+import com.PBA.budgetservice.persistance.model.dtos.IncomeResponse;
+import com.PBA.budgetservice.persistance.model.dtos.IncomeUpdateRequest;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/budget")
```

Comment:
/budget is not needed in my opinion.

Add the /api in application.yml under server.servlet.context-path property to have it by default.